guide/payments/request.md

```diff
@@ -116,52 +108,126 @@ A piece of meta data unique to invoices is an expiration time. This is how long
    image = "/assets/images/guide/payments/request/single-use-payment-request.png"
    retina = "/assets/images/guide/payments/request/single-use-payment-request@2x.png"
    layout = "shadow"
-   caption = "A newly generated single-use Lightning payment request."
+   caption = "A newly generated single-use payment request."
    alt-text = "Screen showing single-use payment request."
    width = 250
    height = 541
 %}
 
 {% include picture.html
-   image = "/assets/images/guide/payments/request/single-use-payment-request-settings.png"
-   retina = "/assets/images/guide/payments/request/single-use-payment-request-settings@2x.png"
+   image = "/assets/images/guide/payments/request/long-expiry.png"
```

Comment:
For this last image, how about moving the expiration to the bottom below the amount? It's not that important to know.